src/hsmm_mvpy/models.py

```diff
@@ -319,7 +370,7 @@ def EM(self, n_events, magnitudes, parameters,  threshold, magnitudes_to_fix=Non
                         #Average channel activity at those points
                         event_values = np.zeros((self.n_trials, self.n_dims))
                         for trial in range(self.n_trials):
-                            time = self.starts[trial]+ np.argmax(eventprobs[:, trial, event])
+                            time = self.starts[trial]+ np.argmax(eventprobs[:, trial, event])+1
```

Comment:
why +1? location?

Reply:
That is a leftover from previous trials, shouldn't be there I will remove